main.py

```diff
@@ -36,48 +37,63 @@
     '--debug', dest='debug', default=False,
     help='Debugging log level'
 )
+parser.add_argument(
+    '--plot', dest='plot',
+    default='', help='Plot to send to ChatGPT to create scenes',
+)
+parser.add_argument(
+    '--out', dest='out',
+    default='', help='Path to the YAML script to write',
+)
 args = parser.parse_args()
 
-if args.script == '':
-    print("Missing script file")
-    sys.exit(1)
 
-valid_text_options = ['chatgpt', 'fake']
-if args.textai not in valid_text_options:
-    print("Please provide a valid option:" +
-          "chatgpt or fake")
-    sys.exit(1)
+def main():
+    if args.plot != '':
```

Comment:
Now that we know that plot was passed, then we need to check if there is a name in `--script-out`